pr-data.csv

```diff
@@ -2254,16 +2254,16 @@ https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop
 https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common,org.apache.hadoop.yarn.server.federation.store.metrics.TestFederationStateStoreClientMetrics.testSuccessfulCalls,OD,,,
-https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router,org.apache.hadoop.yarn.server.router.TestRouterMetrics.testAppsFailedCreated,OD,,,
-https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router,org.apache.hadoop.yarn.server.router.TestRouterMetrics.testAppsFailedKilled,OD,,,
-https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router,org.apache.hadoop.yarn.server.router.TestRouterMetrics.testAppsFailedSubmitted,OD,,,
-https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router,org.apache.hadoop.yarn.server.router.TestRouterMetrics.testAppsReportFailed,OD,,,
-https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router,org.apache.hadoop.yarn.server.router.TestRouterMetrics.testMulipleAppsReportFailed,OD,,,
-https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router,org.apache.hadoop.yarn.server.router.TestRouterMetrics.testSucceededAppsCreated,OD,,,
-https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router,org.apache.hadoop.yarn.server.router.TestRouterMetrics.testSucceededAppsKilled,OD,,,
-https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router,org.apache.hadoop.yarn.server.router.TestRouterMetrics.testSucceededAppsReport,OD,,,
-https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router,org.apache.hadoop.yarn.server.router.TestRouterMetrics.testSucceededAppsSubmitted,OD,,,
-https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router,org.apache.hadoop.yarn.server.router.TestRouterMetrics.testSucceededMultipleAppsReport,OD,,,
+https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router,org.apache.hadoop.yarn.server.router.TestRouterMetrics.testAppsFailedCreated,OD,Opened,https://github.com/apache/hadoop/pull/8133,
```

Comment:
Why open a new PR for each test? Could you still merge several changes into one PR and close some PRs?